src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -16,6 +16,7 @@
 using Microsoft.AspNetCore.ApiVersioning.Swashbuckle;
 using Build = Maestro.Data.Models.Build;
 using Channel = Maestro.Web.Api.v2018_07_16.Models.Channel;
+using Castle.Core.Resource;
```

Comment:
What is this one for?

Reply:
VS keeps doing this...i have no idea why..it just randomly introduces new usings.